app/src/main/java/de/markusfisch/android/shadereditor/widget/ShaderEditor.java

```diff
@@ -803,4 +812,4 @@ public boolean isDone() {
 			return task != null && task.isDone();
 		}
 	}
-}
+}
```

Comment:
I don't know what is up with that line, probably some `\n` or `\r\n` Unicode difference...

Reply:
That is just your editor not finishing the last line with a line break 😉 Which is a convention for text files (on *nix) so they can be dumped on the shell (i.e. `$ cat some_file`) and the prompt will still be in its own line.